src/model_gauge.cpp

```diff
@@ -174,7 +175,7 @@ void ModelGauge::verify_model()
     delay(150);
 
     read_word(0x04, SOC_1, SOC_2);
-    if(SOC_1 >= _config.SOCCheckA && SOC_1 <= _config.SOCCheckB) 
+    if(SOC_1 > _config.SOCCheckA && SOC_1 <= _config.SOCCheckB) 
```

Comment:
I think we tested and concluded that the `>` works and `>=` doesn't.

Reply:
> I think we tested and concluded that the `>` works and `>=` doesn't.

OK. We keep the previous implementation.